hadoop-ozone/dev-support/checks/hadolint.sh

```diff
@@ -17,25 +17,30 @@ DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 cd "$DIR/../../.." || exit 1
 REPO_DIR="$DIR/../../.."
 
-ERROR=0
-
+REPORT_DIR=${OUTPUT_DIR:-"$DIR/../../../target/hadolint"}
+mkdir -p "$REPORT_DIR"
+REPORT_FILE="$REPORT_DIR/summary.txt"
 
 for Dockerfile in $(find "$REPO_DIR/hadoop-ozone" "$REPO_DIR/hadoop-hdds" -name Dockerfile | sort); do
```

Comment:
I suggest the following improvement:

```suggestion
for Dockerfile in $(find hadoop-ozone hadoop-hdds -name Dockerfile | sort); do
```

This would provide cleaner output, eg.:

```
hadoop-ozone/dev-support/docker/Dockerfile
```

instead of

```
/opt/ozone/src/ozone/hadoop-ozone/dev-support/checks/../../../hadoop-ozone/dev-support/docker/Dockerfile
```